.travis.yml

```diff
@@ -23,9 +25,12 @@ services:
   - postgresql
 
 before_install:
-  - (cd ~ && wget http://download.osgeo.org/gdal/2.1.1/gdal-2.1.1.tar.gz -O gdal.tar.gz && tar xzf gdal.tar.gz && cd gdal-2.1.1 && ./configure --with-python && make -j 2 && sudo make install && sudo ldconfig)
+  - sudo apt-get -qq update
 
 install:
+  - export CPLUS_INCLUDE_PATH=/usr/include/gdal
+  - export C_INCLUDE_PATH=/usr/include/gdal
+  - pip install gdal==1.10
```

Comment:
I have already tried... it doesn't work, because postgis gets uninstalled when `libgdal1-dev` is installed. We also need ogr2ogr, which is in the binaries, not in this package.